documentation/modules/adding-source-provider.adoc

```diff
@@ -91,12 +91,19 @@ ifdef::vmware[]
 +
 
 . Choose one of the following options for validating CA certificates:
-
++
 ** *Skip certificate validation* : Migrate without validating a CA certificate.
-** *Use the system CA certificates*: Migrate after validating the system CA certificates.
-
-.. To skip certificate validation, select the *Skip certificate validation* check box.
-.. To validate the system CA certificates, leave the *Skip certificate validation* check box cleared.
+** *Use the system CA certificate*: Migrate after validating the system CA certificate.
```

Comment:
I made sense to list the insecure option first as it was the only option that actually worked but with the latest changes in this area, ut makes more sense to change the order: first, the custom certificate since it would likely be the most commonly used option, second, the system certificate , and lastly the insecure option